### PR TITLE
Add Census Live Syncs Quickstart

### PR DESCRIPTION
**Pull request by dougvalenta**
We created a quickstart for using Dynamic Tables with Census Live Syncs. I accept the Snowflake CLA.

### CODE REVIEW

**Comment by sfc-gh-lambrosetti** [comment]
Hi Doug - the images aren't loading - please make sure the images are formatted correctly in the markdown

example:

`image name`

Otherwise, this looks good!

**Comment by dougvalenta** [comment]
Thanks for taking a look, Luke! The format I used is used in other guides in the repo (e.g. [in "Churn Modeling in Hex"](https://github.com/Snowflake-Labs/sfquickstarts/blob/d50dc6c99bf6ff928e37ff67bb0fcd7988e73799/site/sfguides/src/churn_modeling_in_hex/churn_modeling_in_hex.md?plain=1#L48)), and the images also appear inline when I "View File" in GitHub. I did notice while working on this that images were not loading for _any_ quickstarts when I ran locally. I'm not familiar enough with the dev tooling being used in this project to understand why.

I'm happy to add alt text to the images if you think they need it, but I don't think that's why they aren't loading.

**Comment by iamontheinet** [comment]
@dougvalenta Images not loading locally was an issue on our end at one point. Can you please sync your fork with the upstream master branch and try it again? The issue was the gulp version in `site/package.json` file. If it still doesn't work, then go ahead and create a PR provided everything else is good to go. Thanks!

**Comment by dougvalenta** [comment]
@iamontheinet I'm not able to run locally at all anymore after pulling in latest `master`. No errors in the console other than the sass deprecation warnings, just doesn't end up actually listening on 8080. I'll have to try to get this working when I have more time

File: site/sfguides/src/triggering_real_time_notifications_using_dynamic_tables_and_census_live_syncs/triggering_real_time_notifications_using_dynamic_tables_and_census_live_syncs.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+summary: In this guide, you will learn how to set up a live sync between Census and Snowflake to move data to your activation platforms in real time.
+categories: partner-integrations,marketing
+environments: web
+status: Published 
+feedback link: https://github.com/Snowflake-Labs/sfguides/issues
+tags: Census, Real-Time, Dynamic Tables, Reverse ETL
+
+# A Guide to Triggering Real-Time Notifications Using Dynamic Tables and Census Live Syncs
+
+## Overview
```
Comment (iamontheinet) [inline site/sfguides/src/triggering_real_time_notifications_using_dynamic_tables_and_census_live_syncs/triggering_real_time_notifications_using_dynamic_tables_and_census_live_syncs.md:12]: @dougvalenta @sfc-gh-lambrosetti Add link to Snowflake account signup or login page in under Prereqs or What You'll Need section.

**Review by iamontheinet — CHANGES_REQUESTED** [review]

File: site/sfguides/src/triggering_real_time_notifications_using_dynamic_tables_and_census_live_syncs/triggering_real_time_notifications_using_dynamic_tables_and_census_live_syncs.md (hunk 1 above)
Comment (dougvalenta) [reply site/sfguides/src/triggering_real_time_notifications_using_dynamic_tables_and_census_live_syncs/triggering_real_time_notifications_using_dynamic_tables_and_census_live_syncs.md:12]: Added

**Pull request merged**